### ISSUE TITLE
Keystrokes not being captured

### ISSUE THREAD

**Issue by pirocles** [minetest/minetest#5608]
I'm experiencing an issue on the latest build where any keystrokes I make are sent to the terminal I launched minetest from, not the game itself. The same things happens if I try to launch the game through something like an automator script.

macOS Sierra 10.12.3

**Comment by sfan5** [comment]
and that happens while you are focused on the game window?
also: where did you install from?

**Comment by pirocles** [comment]
Yes, when the game window is focused. I built from commit 6120251.

**Comment by Shalmezad** [comment]
I can confirm I ran into the same issue working on my small patch earlier.
Game window focused, launching from terminal.
Built from commit dc9e451 (`cmake . -DRUN_IN_PLACE=true -DXXF86VM_LIBRARY=/opt/X11/lib`)

**Comment by Shalmezad** [comment]
Did some digging, appears to possibly be an issue with irrlicht through cmake. 
Someone else had a similar issue here: http://irrlicht.sourceforge.net/forum/viewtopic.php?t=36796
Running in fullscreen, keyboard input works fine like the post mentioned.

**Comment by sfan5** [comment]
Please check if this persist in 5.5.0

**Comment by rubenwardy** [comment]
This should also be explicitly states on the docs for building on MacOS

**Comment by Zweihorn** [comment]
@Zughy , @sfan5 
> Running the game from the terminal doesn't work, you need launch it from the `.app`. This is user error (perhaps Irrlicht can be made to raise an error).

I cannot confirm this. At least as new maintainer of the MacPorts [Minetest](https://ports.macports.org/port/minetest/) ports the CLI started MT GUI seems to work as expected.

AFAIK the CLI started MT GUI works with keyboard input as expected on:
- current `minetest` MT 5.6.1_1
- upcoming `minetest56-devel` MT 5.6.1_0 rc1 _(upcoming port, still in review process)_

Tested today. Please feel free to ask if you would to like to learn details.

My five cents.
